### DISCUSSION TITLE
Update-2022-10-26

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#153] [Announcements]
Made a teaser trailer for kons-9. Complete with cheesy epic music.

https://youtu.be/THMzaVDaZP8

**Comment by aykaramba** [comment]
I love this so much.  Especially the epic cheesy music.

**Reply by kaveh808** [reply]
Probably the most epic music used to promote Common Lisp.

**Reply by aykaramba** [reply]
Hehehe!  I mean, I just appreciate the humour of it all.  Love to see people having fun.

**Reply by kaveh808** [reply]
It's open source. We should have fun. And do cool things.

**Reply by foretspaisibles** [reply]
To quote one one of my favourite teachers “Being at work does not have to mean being bored.” To which I fully agree ;-)

**Reply by kaveh808** [reply]
Last night I was imagining doing an "In a world..." trailer.

**Comment by omarbassam88** [comment]
This is Amazing. I think this is the best Common Lisp Application Trailer ever made 😍.

**Comment by ghost** [comment]


**Reply by kaveh808** [reply]
I worked on software for Avatar 2. Back in 2011 when we thought production would start "real soon now".

I've been drawn more to retro CG looks in my advancing years.

**Comment by andres-lowrie** [comment]
The trailer is awesome!!! made by day!... now I just need to learn common-lisp in order to contribute!

**Comment by kayomarz** [comment]
Nice video trailers - I especially liked the one with the piano music - https://www.youtube.com/watch?v=i0CwhEDAXB0

**Reply by kaveh808** [reply]
Thanks. They were fun to make.
